src/Controller/Simulation.java

```diff
@@ -676,6 +682,27 @@ public String setSimulationAway(boolean checked){
         return message;
     }
 
+    /**
+     * checks the state of the away mode and the intruder to make a decision
+     * @return String describing what is the state of the home invasion
+     */
+    public String invadeSimHome() {
+        String message;
+        if(!isAway){
+            message = "Intruders typically only intrude when there is no one in the house.";
+        }
+        else {
```

Comment:
It doesn't look if the checkbox is checked. it's printing on clicked, rather than on checked

Reply:
how boutnow

Reply:
Looks good!